apisix/plugins/proxy-rewrite.lua

```diff
@@ -191,11 +194,17 @@ function _M.check_schema(conf)
     end
 
     if conf.regex_uri and #conf.regex_uri > 0 then
-        local _, _, err = re_sub("/fake_uri", conf.regex_uri[1],
-                                   conf.regex_uri[2], "jo")
-        if err then
-            return false, "invalid regex_uri(" .. conf.regex_uri[1] ..
-                            ", " .. conf.regex_uri[2] .. "): " .. err
+        -- remove this check util 'multipleOf' is supported
```

Comment:
What‘s means？

Reply:
I had the wrong perception of `multipleOf` validator. it only take affect on number type but not length of array. I‘ll fix this later.

Reply:
> What‘s means？

I had the wrong perception of `multipleOf` validator. it only take affect on number type but not length of array.